guardduty_slack/regions.tf

```diff
@@ -8,158 +8,158 @@ module "us-east-1" {
 
```

Comment:
```
#
# Generated by ./regions.tf.sh
# Do not directly modify this file
#
```

Could you modify the shell script?

Then generate this file and commit both (.sh + .tf) files please.